src/patternfly/components/Chip/chip.scss

```diff
@@ -88,3 +106,7 @@
   font-size: var(--pf-c-chip__text--FontSize);
   color: var(--pf-c-chip__text--Color);
 }
+
+.pf-c-chip__icon {
+  margin-right: var(--pf-c-chip__icon--MarginRight);
```

Comment:
I wonder if this should be `__icon + __text, __text + icon { margin-left... }`? That would allow the icon to be placed before or after the text, and also have a standalone `__icon`

Reply:
nicee!